pkg/ebpf/c/bpf_bypass.h

```diff
@@ -48,4 +48,19 @@ static __always_inline typeof(name(0)) ____##name(struct pt_regs *ctx, ##args)
 #define BPF_BYPASSABLE_UPROBE(name, args...)  BPF_BYPASSABLE_KPROBE(name, ##args)
 #define BPF_BYPASSABLE_URETPROBE(name, args...)  BPF_BYPASSABLE_KRETPROBE(name, ##args)
 
+#define BPF_BYPASSABLE_PROG(name, args...)						    \
```

Comment:
can you add a short comment? (also for the other existing macros) to explain them (providing an example where they would be used in 1 short paragraph)

Reply:
For BPF_BYPASSABLE_PROG - done
for the rest - in a separate PR